src/main/kotlin/org/jdc/template/ux/about/AboutViewModel.kt

```diff
@@ -101,6 +107,82 @@ class AboutViewModel
         })
     }
 
+    /**
+     * Individual web service call
+     */
+    fun testIndividualServiceCall() {
```

Comment:
Do this work in the StartupViewModel (on launch) so that there is data when the app first launches

Reply:
@jeffdcamp When I did this task originally I wondered how you expected it to behave.  To make it work in the StartupViewModel I would move it there.  I assume that you are asking me to do that, so I will make that change.